tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -41,6 +44,19 @@
 except ImportError:
     from setup_paths import OPHYS_DATA_PATH, OUTPUT_PATH
 
+skip_on_darwin_arm64 = pytest.mark.skipif(
+    platform.system() == "Darwin" and platform.machine() == "arm64",
+    reason="Tests are skipped on macOS ARM64 because of incompatibility with the 'isx' module",
+)
+skip_if_isx_not_installed = pytest.mark.skipif(
+    not importlib.util.find_spec("isx"),
+    reason="Tests are skipped because the 'isx' module is not installed.",
+)
+skip_on_python_313 = pytest.mark.skipif(
```

Comment:
You can add those test in the top of the test that is skipped just to keep content closer to use.

Also, can you reference the issue to keep track:

https://github.com/inscopix/pyisx/issues